packages/flutter/test/widgets/scrollable_test.dart

```diff
@@ -676,4 +676,144 @@ void main() {
     // of Platform.isMacOS, don't skip this on web anymore.
     // https://github.com/flutter/flutter/issues/31366
   }, skip: kIsWeb);
+
+  testWidgets('Can recommendDeferredLoadingForContext - animation', (WidgetTester tester) async {
+    int cheapWidgets = 0;
+    int expensiveWidgets = 0;
+    final ScrollController controller = ScrollController();
+    await tester.pumpWidget(Directionality(
+      textDirection: TextDirection.ltr,
+      child: ListView.builder(
+        controller: controller,
+        physics: const NeverScrollableScrollPhysics(),
```

Comment:
Why this? Shouldn't this also work if the user is allowed to scroll? (same for the next test)

Reply:
This doesn't do what I thought it did, removed.